ngraph/test/onnx/onnx_import_quant.in.cpp

```diff
@@ -307,27 +307,6 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_quant_conv_linear_3d)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, onnx_model_qlinear_matmul)
```

Comment:
BTW, a lot of tests are removed but the implementation of QLinearMatMul in the onnx_importer remains in the code base. What's the point of removing the tests in this case?

Reply:
@ilyachur, @jane-intel I suppose we can remove OPs which is not opset 1+, like  QLinearMatMul, Dot, QuantizedDot etc., along with removing v0 OPs. What do you think?

Reply:
Yes we have a ticket to remove all legacy operations which are not represented in newer opsets.

Reply:
I don't think these quantized ops should removed. It may be possible to update them to use new v1+ ops.

Reply:
I'm beginning to feel more and more strongly, that this PR should be split into multiple PRs, each addressing a specific area.

Reply:
@postrational even if we will decide to remove these OPs, I'll definitely create separate PR. This one is huge enough already

Reply:
Will be removed in separate PR in the scope of task 31835